server/src/internalClusterTest/java/org/opensearch/search/sort/FieldSortIT.java

```diff
@@ -105,7 +109,24 @@
 import static org.hamcrest.Matchers.nullValue;
 import static org.hamcrest.Matchers.oneOf;
 
-public class FieldSortIT extends OpenSearchIntegTestCase {
+public class FieldSortIT extends ParameterizedOpenSearchIntegTestCase {
```

Comment:
I think we can skip this one as this has some tests failures and I am fixing it as part of https://github.com/opensearch-project/OpenSearch/issues/8510. I am planning to enable the parameterization as part of the pr that contains the fix.

Reply:
makes sense, ack.